ckanext/datagovtheme/templates/package/read.html

```diff
@@ -59,14 +59,14 @@ <h1 itemprop="name" style="margin-bottom:5px;">
   <h3>Access &amp; Use Information</h3>
   <div class="access-use-main">
   {% if h.get_pkg_dict_extra(pkg, 'accessLevel') == 'non-public' %}
-  <span class="access-non-public"><i class="icon-lock"></i><strong><a href="https://project-open-data.cio.gov/v1.1/schema/#accessLevel">Non-public:</a></strong> This dataset is not for public access or use. Specific details are provided below.</span>
+  <span class="access-non-public"><i class="fa fa-lock"></i><strong><a href="https://project-open-data.cio.gov/v1.1/schema/#accessLevel">Non-public:</a></strong> This dataset is not for public access or use. Specific details are provided below.</span>
```

Comment:
href should be https://resources.data.gov/schemas/dcat-us/v1.1/#accessLevel

Not sure if we can stick this in a configuration variable since it's used in a few places.